src/amber/cli/templates/controller.cr

```diff
@@ -3,7 +3,7 @@ require "./field.cr"
 module Amber::CLI
   class Controller < Teeplate::FileTree
     include Amber::CLI::Helpers
-    directory "#{__DIR__}/controller"
+    directory "#{__DIR__}/controller/plain"
```

Comment:
What does plain mean?

Reply:
I was struggling to come up with a good name for it. It’s a controller without a model or resource.  It doesn’t have the restful 7 built in methods.

Can you think of a better name?  Obviously Plain isn’t clear. ;-)